packages/carousel/src/Carousel.js

```diff
@@ -145,7 +147,8 @@ function Carousel({
           {showLeftArrow && (
             <ArrowButton
               data-testid="left-arrow"
-              icon={<SmChevronLeft size={24} />}
+              {...leftButton}
+              icon={leftButton?.icon || <SmChevronLeft size={24} />}
```

Comment:
if we pass props after the default props, we don't have to to this condition `icon={leftButton?.icon || <SmChevronLeft size={24} />}`